src/services/TestResultsService.ts

```diff
@@ -630,6 +633,11 @@ export class TestResultsService {
     }
     return missingMandatoryFields;
   }
+
+  private isWithinTwoMonthsFromToday(date: Date) {
+    return date.toISOString().substring(0, 10) > new Date().toISOString().substring(0, 10) && date.toISOString().substring(0, 10) < dateFns.addMonths(new Date(), 2).toISOString().substring(0, 10);
```

Comment:
Should today be in this range? Would need >=

Reply:
I checked the original ACs when this change was introduced (https://jira.dvsacloud.uk/browse/CVSB-12215) and it excludes today from this range. The previous behaviour was also excluding it so I maintained the logic